src/test-utils/dom/app-layout/index.ts

```diff
@@ -48,6 +49,10 @@ export default class AppLayoutWrapper extends ComponentWrapper {
     return this.findComponent(`.${SplitPanelWrapper.rootSelector}`, SplitPanelWrapper);
   }
 
+  findSplitPanelOpenButton(): ElementWrapper | null {
+    return this.findByClassName(splitPanelStyles['open-button']);
+  }
```

Comment:
This test-util appeared useful in many tests, so I decided to expose it here as well.